docs/container.md

```diff
@@ -171,6 +171,23 @@ printf('Container\'s name is $s', $container->getName());
 printf('Container\'s exit code is %d', $container->getExitCode());
 ```
 
+## Copy files or folders from a container
+
+File and folders can be downloaded as a tar file from the container. The copy() function allows a folder/file to be downloaded as tar stream and copyToDisk() saves the result as a tar file.
```

Comment:
1. [ ] Function, file and directory names should be enclosed with backticks (`)
2. [ ] Add an `s` to the first *File* (first word in paragraph)